uint8 DoorDetection     = 1
uint8 EnterSite         = 2
uint8 CannotOpen        = 3
uint8 GuestRecognition  = 4
uint8 Requesting        = 5
uint8 Leading           = 6
uint8 Introducing       = 7
uint8 Serving           = 8
uint8 Backward          = 9

uint8 NowTask
uint8 NextTask
bool FinishState
bool NeedHelp
